website/src/blogs/2017-02-20-torus-from-coreos-steps-aside-as-cloud-native-storage-platform-what-now.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Torus (from CoreOS) steps aside as Cloud Native Storage Platform. What now?
+author: Kiran Mova
+author_info: Contributor and Maintainer OpenEBS projects. Chief Architect MayaData. Kiran leads overall architecture & is responsible for architecting, solution design & customer adoption of OpenEBS.
+tags: DevOps, Docker, Storage Containers, Kubernetes
+date: 20-02-2017
+excerpt: Torus (from CoreOS), that aimed at providing container-native distributed storage announced that it would discontinue development due to lack of traction.
+---
+
+[Torus (from CoreOS)](https://github.com/coreos/torus), which aimed at providing container-native distributed storage announced that it would discontinue development due to lack of traction. It just goes to show the dynamics of the OpenSource projects and how business needs could influence them.
+
+Torus attempted to solve one of the core infrastructure problems, and like any projects that aim to solve the infrastructure problems — compute, network or storage, are slightly harder problems to crack with only a handful of people really venturing into solving them. Maybe, this inertia is also due to the fact that, when infrastructure fails, hell breaks loose — we are all too familiar with those network outages and data loss situations where Database teams spend days recovering data.
+
+Harder problems require renewed focus, at least in the initial stages to get the product off the ground for the basic use case. The entry barrier must be minimal, if not totally absent. Once the core-value is established, in terms of ease of use or scale and just solving a problem in an intuitive way, it graduates into adoption phase.
+
+With containers changing the landscape of applications, they pose an additional set of requirements on the storage platform, which are covered in my earlier post — [Emerging Storage Trends for Containers](https://blog.openebs.io/emerging-storage-trends-for-containers-4970e4c51de#.ep5wl2u0z). *Any container-native storage platform developed today should be distributed and natively integrated into container orchestration platforms like Kubernetes.* Torus got these aspects right.
```

Comment:
```suggestion
With containers changing the landscape of applications, they pose an additional set of requirements on the storage platform, which are covered in my earlier post — [Emerging Storage Trends for Containers](https://blog.openebs.io/emerging-storage-trends-for-containers-4970e4c51de#.ep5wl2u0z). _Any container-native storage platform developed today should be distributed and natively integrated into container orchestration platforms like Kubernetes_. Torus got these aspects right.
```

Reply:
Single star also provides italic styling to the text
<img width="1440" alt="Screenshot 2021-07-06 at 1 26 19 PM" src="https://user-images.githubusercontent.com/34312966/124563622-d2821f80-de5d-11eb-8881-f4e31be87c9f.png">